### DISCUSSION TITLE
while configuration apiman ui in tomcat getting below error

### DISCUSSION THREAD

**Discussion by shachindrasingh** [apiman/apiman#2533] [General]
<img width="1503" alt="Screenshot 2023-05-22 at 1 08 20 PM" src="https://github.com/apiman/apiman/assets/8200140/f7eb3630-bf57-48ec-ad50-05071cb2da39">

need help in this. i am using apiman 3.1.1 version.

**Comment by msavy** [comment]
You need to set up your own auth with the Tomcat distro. 

It just uses Tomcat's default BASIC auth by default (`tomcat-users.xml`).

If you want to use something else, you need to set it up yourself (e.g. Keycloak, Auth0, etc).

**Reply by shachindrasingh** [reply]
user detail already added
<img width="1503" alt="Screenshot 2023-05-22 at 2 12 57 PM" src="https://github.com/apiman/apiman/assets/8200140/7c5a6fd8-21e7-4f9d-980a-13447329f08a">

**Reply by shachindrasingh** [reply]
@msavy  still i am facing the same issue.

**Reply by msavy** [reply]
What issue, you haven't explained

**Reply by shachindrasingh** [reply]
@msavy  issue is that , i am trying to run apiman ui on tomcat. we had done all required configuration in tomcat. when we enter login and password of apimanui, ideally it open the apimanager screen but system prompt again enter the user id and password.
